.eslintignore

```diff
@@ -0,0 +1 @@
+/dist
```

Comment:
- [Optional] Please consider adding a .gitignore file and add `node_modules` in it so that it will be ignored by git and won't be pushed to your GitHub.